repose-aggregator/src/docs/asciidoc/services/open-tracing.adoc

```diff
@@ -0,0 +1,133 @@
+= OpenTracing Service
+
+The OpenTracing service (http://opentracing.io/) is a service in *Repose* which allows for sending tracing data to a Tracing service.
```

Comment:
Something about the way this link is presented bothers me ever so slightly. I propose something like this:
> The OpenTracing service, which adheres to the http://opentracing.io[OpenTracing standard], enables reporting of tracing data to a collection service.

Of course, that's just my preference, so feel free to use some, all, or none of that.

Reply:
Updated.